README.md

```diff
@@ -0,0 +1,53 @@
+
+![Axoflow logo](AxoFlowBlueBlack.svg)
+
+# Cloud-ready syslog-ng images by Axoflow
+
+This repository contains the cloud-ready syslog-ng images created and maintained by [Axoflow](https://axoflow.com). Our images are different from the [upstream syslog-ng images](https://hub.docker.com/r/balabit/syslog-ng/) in a number of ways:
+
+- They are based on Alpine Linux, instead of Debian testing for reliability and smaller size (thus smaller attack surface).
+- They incorporate cloud-native features and settings (such as the Kubernetes source).
+- They incorporate container-level optimizations (like the use of an alternative malloc library) for better performance and improved security.
+- They support the ARM architecture.
+
+Our images are available for the following architectures:
+
+- amd64
+- arm/v7
+- arm64
+
+## How to use
+
+You can find the list of tagged versions at [https://github.com/axoflow/syslog-ng-docker/pkgs/container/syslog-ng](https://github.com/axoflow/syslog-ng-docker/pkgs/container/syslog-ng).
+
+To install the latest stable version, run:
+
+```shell
+docker pull ghcr.io/axoflow/syslog-ng:latest
+```
+
+You can also use it as a base image in your Dockerfile:
+
+```shell
+FROM ghcr.io/axoflow/syslog-ng:latest
+```
+
+If you want to test a development version, you can use the nightly builds:
+
+```shell
```

Comment:
We should include here which syslog-ng version each of the channels have and that these packages are automatically updated as a new syslog-ng package comes out.

Reply:
Do we have a list of channels somewhere? I've just mentioned these because I saw them on the tags page.